explore-analyze/visualize/charts/bar-charts.md

```diff
@@ -0,0 +1,413 @@
+6. Customize how the annotation appears:
+   - **Name**: The name of the annotation. 
+   - **Icon decoration**: Allows adding an icon next to the label.
+   - **Text decoration**: Selects whether the label shows nothing or the annotation's name.
+   - **Line**: Sets stroke width (1 px) plus pattern (solid or dashed).
+   - **Color**: Sets the line color.
+   - **Hide annotation**: Hides or displays the annotation. By default, this setting is turned off and the annotation is displayed.
+   - **Tooltip**: Adds additional fields to the annotation label. 
+6. Click **Close** to apply the configurations. Your bar chart will now display the annotations at the appropriate time points.
+
+To edit existing annotations click on them in the annotations list. If you no longer need an annotation, you can delete it.
+
+## Bar chart settings [settings]
+
+Customize your bar chart to display exactly the information you need, formatted the way you want.
+
+### Horizontal axis settings [horizontal-axis-options]
+
+**Data**
+:   The dimension that creates your individual bars. Common options include:
+    - **Date histogram**: Create time-based bars with configurable intervals.
+    - **Filters**: Define custom categories using KQL queries.
+    - **Intervals**: Group data into numerical ranges.
+    - **Top values**: Specify fields for which to gather top values. 
+
+**Appearance**
+:   Define the formatting of the horizontal axis, including:
+    - **Name**: Customize the axis label to describe what the bars represent.
+    - **Value format**: Select to display values as number, percent, bytes, bits, duration, or with a custom format.
+
+### Vertical axis settings [vertical-axis-options]
+
+**Data**
+:   The metric that determines the height of your bars. When you drag a field onto the vertical axis, {{kib}} suggests a function based on the field type. You can change it and use aggregation functions like `Sum`, `Average`, `Count`, `Median`, `Min`, `Max`, and more, or create custom calculations with formulas. Refer to [](/explore-analyze/visualize/lens.md#lens-formulas) for examples, or to the {icon}`documentation` **Formula reference** available from Lens.
+
+    :::{include} ../../_snippets/lens-value-advanced-settings.md
+    :::
+
+**Appearance**
+:   Define the formatting of the vertical axis, including:
+    - **Name**: By default, the chart uses the function or formula as the axis label. It's a best practice to customize this with a meaningful title.
+    - **Value format**: Select to display values as number, percent, bytes, bits, duration, or with a custom format.
+    - **Series color**: Assign a specific color to bars.
+    - **Axis side**: Choose to display the veritical axis on the left or right side of the graph. By default, the axis displays on the left. 
+
+### Break down by settings [breakdown-options]
+
+**Data**
```

Comment:
Likewise for this one we should list the additional options available with each of these options